tests/tpl/cli11_smoke_test.cpp

```diff
@@ -0,0 +1,72 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "umpire/tpl/CLI11/CLI11.hpp"
+
+std::string getVersion()
+{
+  std::stringstream sstr;
+
+  sstr << CLI11_VERSION;
+
+  return sstr.str();
+}
+
+//-----------------------------------------------------------------------------
+int main(int argc, char* argv[])
```

Comment:
Might be good to have some gtest assertions in here for stuff that should definitely be true.

Reply:
Agreed.  I need to find some examples someplace of a gtest way of passing in command line arguments to be parsed.  This is presently just being called directly.

Reply:
Done